src/containerapp/azext_containerapp/custom.py

```diff
@@ -2016,6 +2016,11 @@ def containerapp_up(cmd,
         ingress = "external" if not ingress else ingress
         target_port = 80 if not target_port else target_port
 
+    if image:
+        if ingress and not target_port:
+            target_port = 80
+            logger.warning("No ingress provided, defaulting to port 80.")
```

Comment:
Also add the command to change this later. i.e may be containerapp up with --ingress & --target-port & the update command they can run post app is created.

Reply:
@panchagnula should this be logger.info or warning?